app/src/main/res/layout/view_page_header.xml

```diff
@@ -26,6 +26,39 @@
             android:layout_width="match_parent"
             android:layout_height="match_parent" />
 
+        <LinearLayout
```

Comment:
The clickable area is quite small. Would it be better if adds `padding` to the `LinearLayout`?